curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md

```diff
@@ -0,0 +1,262 @@
+
+2. Inside the `.space` `div`, there should be two more `div` elements with the class `earth` and `orbit`, respectively.
+
+3. Inside the `.orbit` `div`, there should be another `div` with the class `moon`.
+
+4. The `space` `div` should be centered on the page and have a width and height of `200px`.
+
+5. The `earth` `div` should be centered inside the `space` `div` and have a width and height of `100px`.
+
+6. The `orbit` `div` should be centered with respect to the earth and have a width and height of `200px`.
+
+7. The orbit path for the moon around the Earth should be a circle. 
+
+8. The moon `div` should be centered inside the `orbit` `div` and have a width and height of `30px`.
+
+9. Your `.earth` and `.moon` `div` elements should have a background color and `border-radius` to make them look like planets.
+
+10. You should define a `@keyframes orbit` animation that rotates the `.orbit` element 360 degrees around its center. You should apply this animation to the `.orbit` element with a duration of `5` seconds, a linear timing function, and infinite iterations.
+
+# --hints--
+
+You should have a main `div` with the class `space`.
+
+```js
+assert(document.querySelector('div')?.getAttribute('class') === 'space');
+```
+
+The `space` `div` should have a width and height of `200px`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.space')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.space')?.height === '200px');
+```
+
+You should have a `div` with the class `earth` inside the `.space` `div`.
+
+```js
+assert.exists(document.querySelector('div.space div.earth'));
+```
+
+The `earth` `div` should have a width and height of `100px`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.earth')?.width === '100px' && new __helpers.CSSHelp(document).getStyle('.earth')?.height === '100px');
+```
+
+You should have a `div` with the class `orbit` inside the `.space` `div`.
+
+```js
+assert.exists(document.querySelector('div.space div.orbit'));
+```
+
+The `orbit` `div` should have a width and height of `200px`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.orbit')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.orbit')?.height === '200px');
+```
+
+You should have a `div` with the class `moon` inside the `.orbit` `div`.
+
+```js
+assert.exists(document.querySelector('div.space div.orbit div.moon'));
+```
+
+(combined with &&) The `moon` `div` should have a width and height of `30px`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.moon')?.width === '30px' && new __helpers.CSSHelp(document).getStyle('.moon')?.height === '30px');
+```
+
+Your `.earth` `div` element should have a background color.
+
+```js
+assert.exists(new __helpers.CSSHelp(document).getStyle('.earth')?.backgroundColor);
+```
+
+Your `.earth` element should have `border-radius` to make it look like a circle.
+
+```js
+assert.exists(new __helpers.CSSHelp(document).getStyle('.earth')?.borderRadius);
```

Comment:
```suggestion
assert(new __helpers.CSSHelp(document).getStyle('.earth')?.borderRadius);
```

Reply:
Also, to make it look like a circle maybe we should check that the value is set to 50%. What do you think?